addons/music/fnc_stopMusic.sqf

```diff
@@ -0,0 +1,25 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_stopMusic
+
+Description:
+    A function used to stop any music playing. Must have been started with CBA_fnc_playMusic, otherwise it is ignored. 
+    
+Parameters:
+    none
+
+Returns:
+    BOOL: true if music was stopped, false if already stopped
```

Comment:
Functions that "do stuff" as opposed to setter functions report usually nothing / `nil` in SQF, but `true` is ok too.